modules/@apostrophecms/task/index.js

```diff
@@ -222,11 +216,24 @@ module.exports = {
             return key;
           }
         };
+        addCloneMethod(req);
         req.res.__ = req.__;
         const { role, ..._properties } = options || {};
         Object.assign(req, _properties);
         self.apos.i18n.setPrefixUrls(req);
         return req;
+
+        function addCloneMethod(req) {
+          req.clone = (properties = {}) => {
+            const _req = {
+              ...req,
+              ...properties
+            };
+            self.apos.i18n.setPrefixUrls(_req);
+            addCloneMethod(_req);
```

Comment:
Why do we need to run this on the req again?

Reply:
Calling addCloneMethod gives us a closure in which the request that will be cloned is the correct request and not the original one returned by `task.getReq`. This becomes important when a request is cloned and then cloned again. We don't want a copy of the original, we want a copy of the first clone, and the ability to make a third clone that copies the second with our own `clone` method, etc.